File: packages/python/examples/credits.py
import os
import asyncio
from ailabtools import AILabClient

async def main():
    api_key = os.environ.get('AILAB_API_KEY')
    if not api_key:
        raise RuntimeError('Missing AILAB_API_KEY')
    client = AILabClient(api_key=api_key)
    credits = await client.common.commonQueryCredits({})
    print('credits:', credits)
    await client.aclose()

if __name__ == '__main__':
    asyncio.run(main())

File: packages/node/examples/credits.ts
import { AILabClient } from '../src/index';

async function main() {
  const apiKey = process.env.AILAB_API_KEY;
  if (!apiKey) throw new Error('Missing AILAB_API_KEY');
  const client = new AILabClient({ apiKey });
  const credits = await client.common.commonQueryCredits({});
  console.log('credits:', credits);
}

main().catch((err) => {
  console.error(err);
  process.exit(1);
});
